### ISSUE TITLE
CheckInterval in wrong section in default cmd/res/configuration.toml

### ISSUE THREAD

**Issue by siggiskulason** [edgexfoundry/device-camera-go#40]
When using device-camera-go in a clean install with no consul settings for device-camera-go and doing

```
make build
./run.sh
```

then then the service fails to start with 

```
 msg="could not register service with Registry: unable to register service with consul: Service information not set"
```

The reason is that the CheckInterval = '15s' setting in cmd/res/configuration.toml is in [Registry] but should be in [Service]. 
This means that the service can't start up as go-mod-registry: Register() fails if client.healthCheckInterval is empty

The solution is to move 
```
[Registry]
...
CheckInterval = '15s'
```

to
```
[Service]
...
CheckInterval = '15s'
```
deleting the data in consul and doing ./run.sh again to update it will then work and the service start.  
I tested this running the core services installed using the geneva/stable edgexfoundry snap.

**Comment by cloudxxx8** [comment]
we are removing check interval very soon

**Comment by siggiskulason** [comment]
Thanks @cloudxxx8  for clarifying that. In that case, this would get automatically resolved when that happens. In the meantime, this is obviously not an issue with production as the toml file would be customised for the actual camera devices being used etc, but just means that device-camera-go doesn't run out of the box as it is for evaluation purposes, with the default settings. I just came across this while reviewing https://github.com/edgexfoundry/device-camera-go/pull/36

**Comment by lenny-intel** [comment]
This has been resolved by #93
